src/DotVVM.Samples.Common/Views/ControlSamples/Repeater/RouteLink.dothtml

```diff
@@ -28,14 +28,14 @@
 		<h3>Client rendered - lowercase</h3>
 		<dot:Repeater DataSource="{value: Pages}" RenderSettings.Mode="Client">
 			<ItemTemplate>
-				<dot:RouteLink routeName="RepeaterRouteLink-PageDetail" Param-id="{value: Id}" text="{value: Name}" />
+				<dot:RouteLink routeName="RepeaterRouteLink-PageDetail" Param-Id="{value: Id}" text="{value: Name}" />
```

Comment:
The check should be case insensitive, as most things in dothtml are.

Reply:
The branch should be `v3-master` - there are already some PRs merged. 
We should delete `master-v3`.